pkg/jsontyper/generator.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"go/format"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/brimsec/zq/cli/inputflags"
+)
+
+func main() {
+	var outName, packageName, varName string
+
+	flag.StringVar(&outName, "o", "", "output filename")
+	flag.StringVar(&packageName, "package", "main", "package name")
+	flag.StringVar(&varName, "var", "tc", "variable name")
+
+	flag.Parse()
+	if flag.NArg() == 0 {
+		flag.Usage()
+		os.Exit(1)
+	}
+	fileName := flag.Arg(0)
+	tc, err := inputflags.LoadJSONConfig(fileName)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "error parsing json config: %s\n", err)
+		return
+	}
+
+	contents := fmt.Sprintf(`package %s
+
+import "github.com/brimsec/zq/zio/ndjsonio"
```

Comment:
Packages in `pkg` shouldn't have any concept of packages outside. Maybe call the package `typegenerator` and put it in `zqd/ingest` or as a sub-package of `zio/ndjsonio`?

Reply:
good point, i'll put it in `zio/ndjsonio`